src/renderer/screens/settings/sections/Accounts/Currencies.js

```diff
@@ -2,34 +2,40 @@
 import React, { useState, useCallback } from "react";
 import { useTranslation } from "react-i18next";
 import { useSelector } from "react-redux";
-import type { CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types";
+import type { Currency, CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types";
 import { cryptoCurrenciesSelector } from "~/renderer/reducers/accounts";
 import TrackPage from "~/renderer/analytics/TrackPage";
 import SelectCurrency from "~/renderer/components/SelectCurrency";
 import Box from "~/renderer/components/Box";
 import { SettingsSectionBody as Body, SettingsSectionRow as Row } from "../../SettingsSection";
 import CurrencyRows from "./CurrencyRows";
 import Track from "~/renderer/analytics/Track";
+import { currencySettingsDefaults } from "~/renderer/reducers/settings";
 
 export default function Currencies() {
   const { t } = useTranslation();
   const currencies = useSelector(cryptoCurrenciesSelector);
-  const [currency, setCurrency] = useState(() => {
-    const btc = currencies.find(c => c.id === "bitcoin");
-    return btc || currencies[0];
-  });
+  const [currency, setCurrency] = useState<CryptoCurrency | TokenCurrency | typeof undefined>();
 
   const handleChangeCurrency = useCallback(
-    (currency: CryptoCurrency | TokenCurrency) => {
+    (currency?: CryptoCurrency | TokenCurrency) => {
       setCurrency(currency);
     },
     [setCurrency],
   );
 
-  return !currency ? null : (
-    <Box key={currency.id}>
-      <TrackPage category="Settings" name="Currencies" currencyId={currency.id} />
-      <Track onUpdate event="Crypto asset settings dropdown" currencyName={currency.name} />
+  const currencyId = currency?.id ?? "placeholder";
+  const currencyName = currency?.name ?? "placeholder";
```

Comment:
why even bother with a placeholder in this case? 
maybe dont add the TrackPage and Track in the template if currency isn't set :shrug: